### PR TITLE
Additional tests for blocklisted breach resolution links

### PR DESCRIPTION
**Pull request by flozia**
Additional tests for the blocked company domain links in the breach resolution as requested and promised in https://github.com/mozilla/blurts-server/pull/2961#discussion_r1170343857.

### CODE REVIEW

File: src/utils/breachResolution.test.js [hunk 1]
```diff
@@ -158,15 +159,61 @@ test('appendBreachResolutionChecklist: data class with a resolution referring to
...
+        DataClasses: [BreachDataTypes.Passwords, BreachDataTypes.SecurityQuestions],
+        Name: 'blockedCompanyName',
+        Domain: 'blockedCompanyName.com'
+      }]
+    }],
+    unverifiedEmails: []
+  }
+
+  appendBreachResolutionChecklist(userBreachData)
+
```
Comment (Vinnl) [inline src/utils/breachResolution.test.js:206]: I assume this needn't be called twice?

```suggestion
```

**Review by Vinnl — APPROVED** [review]
Looking good! Does `t.is` do partial matches? (Since you were able to add the part about the password manager without updating the code.)

File: src/utils/breachResolution.test.js (hunk 1 above)
Comment (flozia) [reply src/utils/breachResolution.test.js]: No, it doesn’t. Leftover from when I had both test cases combined in one.

**Comment by flozia** [comment]
> Looking good! Does `t.is` do partial matches? (Since you were able to add the part about the password manager without updating the code.)

The change originally was introduced in another PR by https://github.com/mozilla/blurts-server/pull/2961/commits/94152f52e3da20813118966c0006e0ea8021a13b. During cherry-picking I must have messed something up and it resulted in another commit hash: https://github.com/mozilla/blurts-server/pull/3000/commits/83f1fb95d935318b1305f52f0fe590db1788fd14.

**Pull request merged**